pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
Let me elaborate in details about line 203. The goal is to let __int64 always be 64 bit.

Before your change
--------------------------
```
#ifdef BIT64  
#define __int64     long  
#else // BIT64  
#define __int64     long long  
#endif // BIT64 
```
For 64bit builds: The code expects BIT64 be defined, so __int64 -> long (64bits on unix).
For 32bit builds: The code expects BIT64 not defined, so __int64 -> long long (64bits).

After your change
------------------------
```
#ifndef BIT64  
#define __int64     long  
#else // BIT64  
#define __int64     long long  
#endif // BIT64
```
If ```BIT64``` is used as original code expected:
For 64bit builds: BIT64 is defined, __int64 -> long long (?? >= 64 bits on unix). -- wrong
For 32bit builds: BIT64 not defined, __int64 -> long (32bits) -- wrong

If ```BIT64``` def is always missing:
For 64bit builds: BIT64 not defined, __int64 -> long (64 bits on unix). -- correct
For 32bit builds: BIT64 not defined, __int64 -> long (32bits) -- wrong

So if I understand correctly, your change only makes 1 scenario correct -- 64bit builds and BIT64 def missing. All other scenarios wrong. With the current code, we use ```BIT64``` def as PAL expected, your change makes it always wrong.